docs/ref/forms/models.txt

```diff
@@ -1,15 +1,257 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+===========
+ModelForms
+===========
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. class:: ModelForm
+
+.. _ref-modelform:
+
+.. currentmodule:: django.forms
+
+:class:`~django.forms.ModelForm` API reference. For introductory material about
+ModelForms, see the :doc:`/topics/forms/modelforms` topic guide.
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` Options
+==========================
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled by setting metadata options as attributes of an inner
+``Meta`` class.
+
+::
+
+    # Example using some sort of Book model
```

Comment:
I was having a look at the [Model _meta API docs](https://docs.djangoproject.com/en/5.1/ref/models/meta/) and was wondering if this should also mention the `_meta` attribute
This would be a suggestion
```suggestion
ModelForm ``_meta`` API
=======================

.. class:: ModelFormOptions

The ``_meta`` API is used to build forms that reflect a Django model. The API
is accessible through the ``_meta`` attribute of each model form, which is an
instance of a ``django.forms.models.ModelFormOptions`` object.

The exact structure of the form produced can be controlled by setting metadata
options as attributes of an inner ``Meta`` class::
```